### PR TITLE
chore: Update RAG E2E test

### PR DESCRIPTION
**Pull request by tomfrenken**
## Context

With this PR, we update the prompt used to test our RAG example code.
The new prompt specifically mentions that the model should showcase class instantiation, which hopefully results in stable tests.

### CODE REVIEW

File: sample-code/src/langchain-azure-openai.ts [hunk 1]
```diff
@@ -95,7 +95,7 @@ export async function invokeRagChain(): Promise<string> {
       Use the following pieces of retrieved context to answer the question.
       If you don't know the answer, just say that you don't know.
       Use ten sentences maximum and keep the answer concise.
-      Also prioritize code examples.
+      Also prioritize code examples, e.g., showing the class instantiaion.
```
Comment (tomfrenken) [inline sample-code/src/langchain-azure-openai.ts:98]:
```suggestion
      Also prioritize code examples, e.g., showing the class instantiation.
```

File: sample-code/src/langchain-azure-openai.ts [hunk 2]
```diff
@@ -95,7 +95,7 @@ export async function invokeRagChain(): Promise<string> {
       Use the following pieces of retrieved context to answer the question.
       If you don't know the answer, just say that you don't know.
       Use ten sentences maximum and keep the answer concise.
-      Also prioritize code examples.
+      Also prioritize code examples, e.g., showing the class instantiation.
```
Comment (KavithaSiva) [inline sample-code/src/langchain-azure-openai.ts:98]: [q] What about: Always include code snippets in your answer including class instantiation.

Comment (tomfrenken) [reply sample-code/src/langchain-azure-openai.ts]: Done

**Review by KavithaSiva — APPROVED** [review]
LGTM

**Pull request merged**